src/EnergyPlus/OutdoorAirUnit.cc

```diff
@@ -1594,7 +1610,7 @@ namespace OutdoorAirUnit {
             if (OutAirUnit(OAUnitNum).OAEquip(CompNum).CoilPlantTypeOfNum == TypeOf_CoilSteamAirHeating) {
                 if (OutAirUnit(OAUnitNum).OAEquip(CompNum).MaxVolWaterFlow == AutoSize) {
                     SimulateSteamCoilComponents(
-                        OutAirUnit(OAUnitNum).OAEquip(CompNum).ComponentName, true, OutAirUnit(OAUnitNum).OAEquip(CompNum).ComponentIndex);
+                        OutAirUnit(OAUnitNum).OAEquip(CompNum).ComponentName, true, OutAirUnit(OAUnitNum).OAEquip(CompNum).ComponentIndex, 1.0, 0.0);
```

Comment:
I'm a little hesitant about what these new magic 1 and 0 are for...

Reply:
@Myoldmopar I could not call an existing function without this magic arguments. Adopted from other similar calls from some where else, I think from UnitarySystem.

Reply:
I'll give you a 1.0 and 0.0 for trying, but delete those and figure something else out.

Reply:
I have spent enough time it figure it out but no success. Finally I tried tried mimic what is done else where.

Reply:
@Myoldmopar @rraustad The problem is that we did not have a check for optional variable QCoilReq [here](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/SteamCoils.cc#L233-L236) in order to avoid the magic numbers 1.0 and 0.0. I added the following check for QCoilReq and replaced it with local variable and resolved the problem. 
        if (present(QCoilReq)) {
            QCoilReqLocal = QCoilReq;
        } else {
            QCoilReqLocal = 0.0;
        }
I just pushed an updated branch.